### ISSUE TITLE
Cannot peronalize tags on inputs per-plugin instance

### ISSUE THREAD

**Issue by sunbit** [influxdata/telegraf#662]
I tried to add custom per-plugin tags, with the syntax
```
[[inputs.exec]]
  command = "/var/graphs/telegraf/odenergy.sh"
  name_override = "consumption"
  interval = 60

   [inputs.exec.tags]
   type = "power"
```

And it works, but if i try to set a tag on a second instance of the same plugin for example:

```
[[inputs.exec]]
  command = "/var/graphs/telegraf/odenergy.sh"
  name_override = "consumption"
  interval = 60

   [inputs.exec.tags]
   type = "power"

[[inputs.exec]]
  command = "/var/graphs/telegraf/odenergy.sh"
  name_override = "consumption_2"
  interval = 10

   [inputs.exec.tags]
   type = "power2"
```

I get an error, pointing at the line defining the second .tags section:

`toml: line 123: table `inputs.snmp.tags' is in conflict with normal table in line 106`

Is possible to accomplish this in some way?

**Comment by sparrc** [comment]
hmm, this is supposed to be valid toml, I'll investigate

**Comment by sparrc** [comment]
it looks like our underlying toml library has a bug, so I created an issue: https://github.com/naoina/toml/issues/18

I'm going to try to migrate to https://github.com/pelletier/go-toml, that library appears to parse the toml correctly.

**Comment by sparrc** [comment]
@sunbit Due to the nature of dynamically applying the config to structs, it would be a huge amount of work to port to the pelletier/go-toml library.

Instead I'm going to workaround this by renaming the global `[tags]` section to `[global_tags]`, it'll be available in 0.10.3.

@sunbit for the time being, if you're not using the global `[tags]` table, you can remove that section from your config file and it should parse OK.

Thanks for the report!
